addons/account/views/report_invoice.xml

```diff
@@ -127,7 +127,7 @@
 
                     <div class="clearfix">
                         <div id="total" class="row">
-                            <div t-attf-class="#{'col-4' if report_type != 'html' else 'col-sm-7 col-md-5'} ml-auto">
+                            <div class="col-5 ml-auto">
```

Comment:
I'll do it qdp style:
"No"

And the reason is that the way you are doing it is by always giving the size of 5 columns for that subtotal section, even on small devices, which means that on small devices this section will take less than half the screen where previously it was taking more than half the screen (for those viewing the invoice from their portal) Also I'm not sure that this is a good fix as the problem will be exactly the same with larger amount. If we go to a country where we pay using millions and that the tax has a long name it will still happen. And for that kind of case, I don't really see the problem of splitting the line on two lines as it is already separated visually by a drawn line (also it seems readable to me). Therefore if we accept that kind of behavior for large amounts there should be no reason to change anything here. Where does the feedback comes from?

Reply:
@csnauwaert I was pretty sure you will say something like that. This feedback comes from AVW of course!: 

"Possible de fixer asap en 13.0 ? [Invoice report] ??" 
https://drive.google.com/a/odoo.com/file/d/1mjeTvMNuBx8BX6UKVYJXmVj83_-4m_by/view?usp=drivesdk